android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -470,6 +472,8 @@ public Map getExportedCustomDirectEventTypeConstants() {
       "injectJavaScript", COMMAND_INJECT_JAVASCRIPT,
       "loadUrl", COMMAND_LOAD_URL
     );
+    map.put("requestFocus", COMMAND_FOCUS);
```

Comment:
You don't need to store mapbuilder to a variable, you can just add it above

Reply:
Sorry, I'm not sure how.
Cause MapBuiler is 7 items max.
![image](https://user-images.githubusercontent.com/10772794/60014417-a71ea480-9689-11e9-9590-584a76f89b2c.png)

Reply:
Can someone help with mapbuilder fix?